examples/resource_lacework_resource_group_lw_account/main.tf

```
@@ -0,0 +1,17 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+provider "lacework" {
+  organization = true
+}
+
+resource "lacework_resource_group_lw_account" "example" {
+  name            = var.resource_group_name
+  description     = var.description
+  accounts        = ["tech-ally"]
```

could we test changing the accounts from this to
```suggestion
  accounts        = ["tech-ally", "mini-ally"]
```
just to verify it applies those changes